bindings/matrix-sdk-ffi/src/platform.rs

```diff
@@ -245,12 +231,66 @@ pub struct TracingFileConfiguration {
     max_files: Option<u64>,
 }
 
+#[derive(PartialEq, PartialOrd)]
+enum LogTarget {
+    Hyper,
+    MatrixSDKFFI,
+    MatrixSDKClient,
+    MatrixSDKCrypto,
+    MatrixSDKCryptoAccount,
+    MatrixSDKOIDC,
+    MatrixSDKHTTPClient,
+    MatrixSDKSlidingSync,
+    MatrixSDKBaseSlidingSync,
+    MatrixSDKUITimeline,
+    MatrixSDKEventCache,
+    MatrixSDKEventCacheStore,
```

Comment:
Let's capitalize only the first letter of acronyms: `MatrixSdkFfi`, `MatrixSdkOidc`, and so on and so forth.

Otherwise it seems that `MatrixSDKOIDC` is yelling at me.

Reply:
Fair enough 👍